### DISCUSSION TITLE
Suggestions for the new Followers feature

### DISCUSSION THREAD

**Discussion by Blazekickn** [RetroAchievements/RAWeb#1073] [Ideas]
- add a section on the followers page for people that are following you but you aren't following them, to make it easier to follow people back
- add a "Follows You" text on people's profiles when they are following you, or bring back the old system (renamed appropriately obviously)
- rename mutual followers to friends

I think these minor changes would make the update a lot less controversial

**Comment by Jamiras** [comment]
> add a "Follows You" text on people's profiles when they are following you, or bring back the old system (renamed appropriately obviously)

The old system just changed "Add Friend" to "Confirm Friend Request" if the user was already following you. I don't think "Confirm Following" makes any sense, and "Become Friends" or something similar doesn't really fit with the feature/functionality.

In fact, that was the only indication _anywhere_ in the system where you could see if someone was following you. There was no centralized location to see everyone who was following you. You had to visit a profile and specifically look at the "Add Friend" command to see if you were being followed.
 
Adding some text indicating that they're following you seems reasonable.

**Reply by Blazekickn** [reply]
yeah i figured you wouldn't really want to change the text, which is why i suggested either

**Comment by Jamiras** [comment]
> add a section on the followers page for people that are following you but you aren't following them, to make it easier to follow people back

Would it be enough to just add a "Follow User" link to Followers that you're not already following? I don't really like breaking the "Followers" list into multiple groups. You could argue that the "Following" section could also be broken up into "Following" and "Mutually Following".

**Reply by Blazekickn** [reply]
yes that's perfectly fine, just something to make it easier to identify

**Comment by Jamiras** [comment]
> rename mutual followers to friends

There is no Mutual Followers section to rename (unless Following or Followers gets split). And even if you are mutually following each other, friends implies a closer bond that our system doesn't support (like online/offline notifications).

**Reply by Blazekickn** [reply]
> There is no Mutual Followers section to rename

i know it doesn't exist, i was just asking if it could. I probably could have picked a better term to use for it than rename. I just think there should be some kind of difference between only following one way and both people following each other

> friends implies a closer bond that our system doesn't support (like online/offline notifications).

It didn't before, i don't really think it would now.

**Comment by ghost** [comment]
I would suggest replacing the private list of followers with just the public number of followers on user's profile, as well as removing e-mail notifications.
